CharlieBackend.Data/Repositories/Impl/DashboardRepository.cs

```diff
@@ -104,7 +104,7 @@ public async Task<List<StudentVisitDto>> GetStudentsPresenceListByStudentIds(IEn
                     .Any(x => studentIds.Contains(x.Student.Id)))
                     .Select(x => new StudentVisitDto
                     {
-                        CourseId = x.Lesson.StudentGroup.CourseId,
+                        CourseId = (long)x.Lesson.StudentGroup.CourseId,
```

Comment:
Why do we have nullable courseId? please make this field as non-nullable in db and in code

Reply:
Its from merged dev branch i think. It will be done in other task as you instructed.